package.json

```
@@ -1,12 +1,14 @@
 {
-  "name": "EngageSphere-Test-Design-Masterclass-turma-2",
+  "name": "engagesphere-test-design-masterclass-turma-2",
   "version": "1.0.0",
   "description": "Sample project with a Node.js backend and a React frontend.",
   "scripts": {
     "install:frontend": "cd frontend && npm install",
     "install:backend": "cd backend && npm install",
     "start:frontend": "cd frontend && npm start",
-    "start:server": "cd backend && npm start"
+    "start:server": "cd backend && npm start",
+    "test": "npx cypress run",
```

O `npx` não é necessário dentro do arquivo `package.json`, visto que o npm já sabe onde procurar os binários para execução.

```suggestion
    "test": "cypress run",
```